R/zchunk_L111.nonghg_en_R_S_T_Y.R

```diff
@@ -240,18 +249,22 @@ module_emissions_L111.nonghg_en_R_S_T_Y <- function(command, ...) {
       # need to filter here to replicate behavior of original aggregate()
       filter(!is.na(supplysector), !is.na(subsector), !is.na(stub.technology)) %>%
       group_by(GCAM_region_ID, supplysector, subsector, stub.technology, year) %>%
-      summarise(energy = sum(value)) ->
+      summarise(energy = sum(value))  %>%
+      ungroup ->
       L101.in_EJ_R_en_S_F_Yh.mlt
-
+#MATCHes up TO HERE
     # Calculate emission shares by energy totals.
 
     L111.nonghg_tg_R_en_S_F_Yh %>%
       left_join(L101.in_EJ_R_en_S_F_Yh.mlt, by = c("GCAM_region_ID", "supplysector", "subsector", "stub.technology", "year")) %>%
       mutate(value = value / energy) %>%
+      ungroup %>%
       select(-energy) %>%
       replace_na(list(value = 0)) ->
       L111.nonghg_tgej_R_en_S_F_Yh
 
+    #EVERYTHING MATCHES, even after spreading to wide format!
```

Comment:
if everything matches, can you go ahead and make a few figures comparing the old and the new dataframes?